hawtio-web/src/main/webapp/app/jvm/js/connect.ts

```diff
@@ -163,7 +163,7 @@ module JVM {
 	var autoconnect = $location.search();
 	if (typeof autoconnect != 'undefined' && typeof autoconnect.name != 'undefined') {	
 		var conOpts = Core.createConnectOptions({
-		        scheme: 'http',
+		        scheme: autoconnect.scheme,
```

Comment:
I wonder if we need to set http as fallback if the scheme is not configured on autoconnect? Or is it always configured?